conf/modules/allele_specific_calling.config

```diff
@@ -33,27 +33,42 @@ process {
     withName: '.*ALLELE_SPECIFIC_CALLING:BCFTOOLS_MERGE' {
         ext.args = '--output-type z'
         ext.prefix = { "${meta.id}" }
-        publishDir = [
-            path: { "${params.outdir}/bootstrapann" },
-            mode: params.publish_dir_mode,
-            saveAs: { filename -> filename.equals('versions.yml') ? null : filename },
-        ]
     }
 
     withName: '.*ALLELE_SPECIFIC_CALLING:RENAME_FILES' {
         ext.prefix = { "${meta.id}" }
+    }
+
+    withName: '.*ALLELE_SPECIFIC_CALLING:SPLIT_MULTIALLELICS' {
+        ext.prefix = { "${meta.id}_norm" }
+        ext.args = '--output-type z --multiallelics -both'
+    }
+
+    withName: '.*ALLELE_SPECIFIC_CALLING:REMOVE_DUPLICATES' {
+        ext.args = '--output-type z --rm-dup none'
+        ext.prefix = { "${meta.id}_split_rmdup" }
+    }
+
+    withName: '.*ALLELE_SPECIFIC_CALLING:ADD_VARCALLER_TO_BED' {
+        ext.args2   = '-s 1 -b 2 -e 3'
+    }
+
+    withName: '.*ALLELE_SPECIFIC_CALLING:BCFTOOLS_ANNOTATE' {
+        ext.args = "-c CHROM,FROM,TO,FOUND_IN --output-type z"
+        ext.prefix = { "${meta.id}_split_rmdup_info" }
```

Comment:
Let's remove this part since we now add the variant caller in the call_variants subworkflow

Reply:
Done!